README.md

```diff
@@ -7,6 +7,11 @@ IndexedDB wrapper for Vuejs based on Dexie
   ``` bash
   npm install vue-idb --save
   ```
+## Exemples
```

Comment:
this example is good for an Vue project, but not good for an nodejs package project